openslides/core/views.py

```diff
@@ -447,6 +461,68 @@ def set_scroll(self, request, pk):
             scroll=request.data)
         return Response({'detail': message})
 
+    @detail_route(methods=['post'])
+    def control_blank(self, request, pk):
+        """
+        REST API operation to blank the projector.
+
+        It expects a POST request to
+        /rest/core/projector/<pk>/control_blank/ with a value for blank.
+        """
+        if not isinstance(request.data, bool):
+            raise ValidationError({'detail': 'Data must be a bool.'})
+
+        projector_instance = self.get_object()
+        projector_instance.blank = request.data
+        projector_instance.save()
+        message = "Setting 'blank' to {blank} was successful.".format(
+            blank=request.data)
+        return Response({'detail': message})
+
+    @detail_route(methods=['post'])
+    def broadcast(self, request, pk):
+        """
+        REST API operation to (un-)broadcast the given projector.
+        This method takes care, that all other projectors get the new requirements.
+
+        It expects a POST request to
+        /rest/core/projector/<pk>/broadcast/ without an argument
+        """
+        if config['projector_broadcast'] == 0:
+            config['projector_broadcast'] = pk
+            projector_broadcast_data(pk)
```

Comment:
One question: Is the autoupdate triggered if there is a request to `/broadcast/`? Now I trigger the autoupdate with this method. I can implement this to `need_full_update_for()`, but does the server recognize, if the need_full_update status change?